src/test/java/io/jenkins/plugins/grading/CoverageScoreTests.java

```diff
@@ -14,15 +12,74 @@
-                .setMissedImpact(-2)
-                .build();
-        CoverageScore coverageScore = new CoverageScore(coverageConfiguration, Ratio.create(99, 100));
+    void shouldCalculateTotalImpactWithZeroCoveredImpact() {
+        final CoverageConfiguration coverageConfiguration = createCoverageConfiguration(-2, 0);
+        final CoverageScore coverageScore = new CoverageScore(coverageConfiguration, Ratio.create(99, 100));
 
         assertThat(coverageScore).hasTotalImpact(-2);
     }
+
+    @Test
+    void shouldCalculateTotalImpactWithZeroMissedImpact() {
+        final CoverageConfiguration coverageConfiguration = createCoverageConfiguration(0, 5);
+        final CoverageScore coverageScore = new CoverageScore(coverageConfiguration, Ratio.create(99, 100));
+
+        assertThat(coverageScore).hasTotalImpact(495);
+    }
+
+    @Test
+    void shouldCalculateTotalImpact() {
+        final CoverageConfiguration coverageConfiguration = createCoverageConfiguration(-1, 3);
+        final CoverageScore coverageScore = new CoverageScore(coverageConfiguration, Ratio.create(99, 100));
+
+        assertThat(coverageScore).hasTotalImpact(296);
+    }
+
+    @Test
+    void shouldGetId() {
+        final CoverageConfiguration coverageConfiguration = new CoverageConfigurationBuilder().build();
+        final CoverageScore coverageScore = new CoverageScore(coverageConfiguration, Ratio.create(99, 100));
+
+        assertThat(coverageScore.getId()).isEqualTo("Line");
+    }
+
+    @Test
+    void shouldGetTotalImpact() {
+        final CoverageConfiguration coverageConfiguration = createCoverageConfiguration(1, 1);
+        final CoverageScore coverageScore = new CoverageScore(coverageConfiguration, Ratio.create(99, 100));
+
+        final int missedImpact = coverageConfiguration.getMissedImpact() * coverageScore.getMissedSize();
+        final int coveredImpact = coverageConfiguration.getCoveredImpact() * coverageScore.getCoveredSize();
+        assertThat(coverageScore.getTotalImpact()).isEqualTo(missedImpact + coveredImpact);
+    }
+
+    @Test
+    void shouldGetCoveredSize() {
+        final Ratio codeCoverageRatio = Ratio.create(99, 100);
+        final CoverageConfiguration coverageConfiguration = new CoverageConfigurationBuilder().build();
+        final CoverageScore coverageScore = new CoverageScore(coverageConfiguration, codeCoverageRatio);
+
+        assertThat(coverageScore.getCoveredSize()).isEqualTo(codeCoverageRatio.getPercentage());
+    }
+
+    @Test
+    void shouldGetMissedSize() {
+        final Ratio codeCoverageRatio = Ratio.create(99, 100);
+        final CoverageConfiguration coverageConfiguration = new CoverageConfigurationBuilder().build();
+        final CoverageScore coverageScore = new CoverageScore(coverageConfiguration, codeCoverageRatio);
+
+        assertThat(coverageScore.getMissedSize()).isEqualTo(100 - codeCoverageRatio.getPercentage());
```

Comment:
Test löschen und diese Assertion oben in `shouldGetProperties` reinnehmen: Es genügt auch:
```suggestion
        assertThat(coverageScore.getMissedSize()).isEqualTo(1);
```

Reply:
Erledigt. 
Ich habe `100 - codeCoverageRatio.getPercentage()` beibehalten, weil es im Test klarer macht, dass die Ratio das Ergebnis von `missedSize` beeinflusst. Ich ändere es aber gerne, wenn sie hard gecoded besser finden.